src/hooks/usePromise.js

```diff
@@ -13,8 +13,6 @@ export default function usePromise(fn, memoParams, defaultValue) {
     return () => {
       cancelled = true
     }
-
-    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I saw in a few places that we remove this rule without adding a global. Is it a safe assumption we are making for those cases?

Reply:
This eslint rules is something it should not be global, since the correct way is to put all dependencies in the [], so if for some reason you want to override you must add this eslint disable line.
https://reactjs.org/docs/hooks-rules.html